### PR TITLE
Added profile pictures

### PR DESCRIPTION
**Pull request by vocatwk**
The page looks a little awkward when there's not that much information to display. Let me know if you guys have any suggestions or changes.

Also I forgot to mention that you only get a badge next to the conversation with the number of messages not yet seen only if you've ever opened the conversation before.
 
User1's view
screen shot 2018-08-03 at 1 51 25 am
User4's view
screen shot 2018-08-03 at 1 55 07 am
screen shot 2018-08-03 at 1 52 21 am
User1's view
screen shot 2018-08-03 at 1 51 52 am
screen shot 2018-08-03 at 1 52 14 am

### CODE REVIEW

**Comment by jasminechau2** [comment]
I think it looks good

File: src/main/webapp/WEB-INF/view/chat.jsp [hunk 1]
```diff
@@ -276,20 +276,27 @@ String membersOfConversation = (String) request.getAttribute("membersOfConversat
...
+              }
+
               if(user.equals(author)){
           %>
                 <div class="messageContainer right">
                   <div class="box triangle rightBubble blue">
                     <%= message.getContent() %>
                   </div>
-                  <img class="dot" src="/default-user.png">
+                  <img class="dot" src="<%= picture %>">
```
Comment (lealgamb) [inline src/main/webapp/WEB-INF/view/chat.jsp]: any chance we can add a link on the image so the logged in user can access his/her profile from the chat page

**Comment by lealgamb** [comment]
LGTM. Perhaps adding Bootstrap's `<ul>` with the class `list-group` can make the messages stand out more.  If you want I can work on this since I have to update the profiles to make sure messages only appear if the logged in user is part of the conversation.

File: src/main/webapp/WEB-INF/view/profile.jsp [hunk 2]
```diff
@@ -22,33 +23,106 @@ ConversationStore conversationStore = ConversationStore.getInstance();
...
-    <% } %>
+    <div id="aboutSection">
+      <h2> About <%= subject %> </h2>    
+
+      <% if (user.equals(subject)) { %>
+        <p> Edit your About Me </p>
+        <form action="/profile/<%= subject %>" method="POST">
+          <input type="text" name="aboutMe">
+          <br/>
+          <button type="submit">Submit</button>
```
Comment (lealgamb) [inline src/main/webapp/WEB-INF/view/profile.jsp]: adding some `margin-top` to the button would be nice.

File: src/main/webapp/WEB-INF/view/chat.jsp (hunk 1 above)
Comment (vocatwk) [reply src/main/webapp/WEB-INF/view/chat.jsp]: Sure, I can try and do that.

File: src/main/java/codeu/controller/ProfileServlet.java [hunk 3]
```diff
@@ -135,4 +135,29 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
...
+      throws IOException, ServletException {
+
+    String requestUrl = request.getRequestURI();
+    String username = (String) request.getSession().getAttribute("user"); 
+    User subject = userStore.getUser(requestUrl.substring("/profile/".length()));
+
+    if(!username.equals(subject.getName())){
+      // user is trying to edit another user's profile
+      // TODO respond with access denied
+      response.sendRedirect("/login");
```
[the remark this answers is not in the document]
Comment (vocatwk) [reply src/main/java/codeu/controller/ProfileServlet.java]: Oh yeah, I forgot about that. Will do!

**Pull request merged**